marlowe-playground-client/src/Monaco.js

```diff
@@ -55,12 +55,6 @@ exports.onDidChangeContent_ = function (editor, handler) {
 }
 
 exports.addExtraLibsJS_ = function (monaco) {
-  monaco.languages.typescript.typescriptDefaults.addExtraLib(
-    require('!!raw-loader!../node_modules/bignumber.js/bignumber.d.ts').default, 'inmemory://model/bignumber.js.d.ts'
-  );
-  monaco.languages.typescript.typescriptDefaults.addExtraLib(
-    require('!!raw-loader!src/Language/Javascript/MarloweJS.ts').default, "inmemory://model/marlowe-js.d.ts"
-  );
```

Comment:
we can't just get rid of these, they need to be set somewhere else if they are going to be removed from here

Reply:
Ugh, sorry David - that shouldn't have been committed yet. :-o

Reply:
Fixed.